packages/express/ami/ami-config.sh

```diff
@@ -13,7 +13,7 @@ f () {
         sleep 5
     done
 
-    until docker top $(docker ps | grep _openemr | cut -f 1 -d " ") | grep httpd &> /dev/null
+    until docker top $(docker ps | grep -- -openemr | cut -f 1 -d " ") | grep httpd &> /dev/null
```

Comment:
```suggestion
    until docker top $(docker ps | awk '/-openemr/{ print $1 }') | grep -qF httpd
```

Although if `jq` is available, `docker ps --format json` could be even cleaner.